### ISSUE TITLE
Add "Check for update" from toolbar menu

### ISSUE THREAD

**Issue by wbadry** [arduino/Arduino#5730]
Although this feature is already implemented in boards and libraries, the compiler itself doesn't have such a mechanism. I'm aware that compiler checks for new version at certain time "maybe at startup", but it would be nice to allow user to check for newer version "especially for stand alone version or night build"

**Comment by per1234** [comment]
I believe you mean "IDE" rather than "compiler".

It's a common convention to do this in **Help > About**.

**Comment by wbadry** [comment]
I would agree with this. It is quite interesting to hold the possibility to force check for new version if exists.
